modules/metastorage/src/integrationTest/java/org/apache/ignite/internal/metastorage/server/raft/ItMetaStorageRaftGroupTest.java

```diff
@@ -224,7 +224,6 @@ public void afterTest() {
      * @throws Exception If failed.
      */
     @Test
-    @Disabled("https://issues.apache.org/jira/browse/IGNITE-22891")
```

Comment:
Can you please explain, what was the reason for the test failure?

Reply:
- oldLeaderId was initialized without refresh leader, sometimes wrong node was stopped
- the logic with waiting for desired count of replicators started/stopped was unstable and led to that the resultFuture was never completed

Reply:
Yep, makes sense. Thanks!